src/devices/Is31fl3730/Is31fl3730.cs

```diff
@@ -0,0 +1,381 @@
+            buffer[logicalRow] = UpdateByte(buffer[logicalRow], mask, value);
+
+            UpdateMatrixRegister(matrix);
+        }
+
+        /// <summary>
+        /// Read value of LED for matrix.
+        /// </summary>
+        /// <param name="matrix">The matrix to use.</param>
+        /// <param name="x">The x dimension for the LED.</param>
+        /// <param name="y">The y dimension for the LED.</param>
+        public int ReadLed(int matrix, int x, int y)
+        {
+            int row = matrix is 0 ? y : x;
+            int column = matrix is 0 ? x : y;
+            int mask = 1 << column;
+            byte[] m = _buffers[matrix];
+            int r = m[row];
+            return r & mask;
+        }
+
+        /// <summary>
+        /// Update decimal point for matrix.
+        /// </summary>
+        public void UpdateDecimalPoint(int matrix, int value)
+        {
+            int matrixOneMask = 128;
+            int matrixtwoMask = 64;
+
+            int mask = matrix is 0 ? matrixOneMask : matrixtwoMask;
+            int row = matrix is 0 ? 6 : 7;
+            byte[] buffer = _buffers[matrix];
+
+            buffer[row] = UpdateByte(buffer[row], (byte)mask, value);
+        }
+
+        /// <summary>
+        /// Fill all LEDs with value.
+        /// </summary>
+        public void FillAll(int value)
+        {
+            foreach (int i in Range(0, 2))
+            {
+                if (_enabled[i])
+                {
+                    Fill(i, value);
+                }
+            }
+        }
+
+        /// <summary>
+        /// Fill all LEDs with value, per Matrix.
+        /// </summary>
+        public void Fill(int matrix, int value)
+        {
+            _buffers[matrix].AsSpan().Fill((byte)value);
+            UpdateMatrixRegister(matrix);
+        }
+
+        /// <summary>
+        /// Reset device.
+        /// </summary>
+        public void Reset() => Write(RESET_REGISTER, EIGHT_BIT_VALUE);
+
+        /// <summary>
+        /// Set the shutdown mode.
+        /// </summary>
+        /// <param name="shutdown">Set the showdown mode. `true` sets device into shutdown mode. `false` sets device into normal operation.</param>
+        public void Shutdown(bool shutdown)
+        {
+            // mode values
+            // 0 = normal operation
+            // 1 = shutdown mode
+            if (shutdown)
+            {
+                _configurationValue |= SHUTDOWN;
+            }
+            else
+            {
+                _configurationValue &= ~SHUTDOWN;
+            }
+
+            WriteUpdateRegister();
+        }
+
+        /// <inheritdoc/>
+        public void Dispose()
+        {
+            _i2cDevice?.Dispose();
+            _i2cDevice = null!;
+        }
+
+        private void Write(byte address, byte[] value)
+        {
+            Span<byte> data = stackalloc byte[value.Length + 1];
+            data[0] = address;
+            value.CopyTo(data[1..]);
```

Comment:
I guess you have to adjust the imports to add the new range (I love them btw :-))

Reply:
Is there guidance on how to make this work for .NET Standard 2.0? I didn't find that.

Reply:
I went with this: https://github.com/dotnet/iot/pull/1930/commits/d82f7ff6fe4e67c4401537151595da18840315a6